### PR TITLE
Dev Final version for testing

### PR DESCRIPTION
**Pull request by carioca-au**
- Make the process works with 1 message per time
- removed unnecessary functions

### CODE REVIEW

**Review by pindge — APPROVED** [review]

File: dags/landsat-scenes-sync/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# DEAfrica Landsat Scenes Sync
+Dags to read, either in bulk or daily, stacs released by USGS and migrate to Africa's servers
```
Comment (alexgleith) [inline dags/landsat-scenes-sync/README.md:2]:
```suggestion
Dags to identify USGS Collection 2 Landsat scenes either in bulk or per-day, push them onto a queue, and then to copy the data to Cape Town and upgrade the STAC metadata and create a notification.
```

File: dags/landsat-scenes-sync/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+## Development
+
+### DAGs
+- landsat-scenes-bulk
+  * This Dag will start a Python process which will download GZ files, which are split per landsat and collection, from USGS
```
Comment (alexgleith) [inline dags/landsat-scenes-sync/README.md:8]:
```suggestion
  * This Dag will start a Python process which will download per-satellite tar.gz files from USGS'
```

File: dags/landsat-scenes-sync/README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+# DEAfrica Landsat Scenes Sync
+Dags to read, either in bulk or daily, stacs released by USGS and migrate to Africa's servers
+
+## Development
+
+### DAGs
+- landsat-scenes-bulk
+  * This Dag will start a Python process which will download GZ files, which are split per landsat and collection, from USGS 
+    file server, send the stac, in bulk, to our SQS queue in Africa.
```
Comment (alexgleith) [inline dags/landsat-scenes-sync/README.md:9]:
```suggestion
    file server, send the STAC documents in bulk to our SQS queue in Africa. 
```

File: dags/landsat-scenes-sync/README.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+## Development
+
+### DAGs
+- landsat-scenes-bulk
+  * This Dag will start a Python process which will download GZ files, which are split per landsat and collection, from USGS 
+    file server, send the stac, in bulk, to our SQS queue in Africa. 
+    
+- landsat-scenes-daily
+  * This Dag will start a Python process which will access the USGS API, accordingly to informed dates, and 
+    send the stac to our SQS queue in Africa.
```
Comment (alexgleith) [inline dags/landsat-scenes-sync/README.md:13]:
```suggestion
  * This Dag will start a Python process which will access the USGS API, accordingly to informed dates, and 
    send the STAC documents to our SQS queue in Africa.
```

File: dags/landsat-scenes-sync/README.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+        
+- landsat-scenes-process
+  * This Dag will be started by the scheduler to pull messages from the SQS queue, validate the message, process 
+    the message adding missing items, changing links to point to our S3 bucket and transforming from stac 0.7 to 1.0. 
+    After that sending to an SNS topic.
```
Comment (alexgleith) [inline dags/landsat-scenes-sync/README.md:18]:
```suggestion
  * This Dag will be started by the scheduler to pull messages from the SQS queue, validate the message, process 
    the message adding missing items, changing links to point to our S3 bucket and transforming from stac 0.7 to 1.0. 
    After that it sends the STAC document to an SNS topic.
```

File: dags/landsat-scenes-sync/README.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+  
+- landsat-scenes-indexing
+  * This Dag will be started by the scheduler to pull the messages from a second SQS queue, which is 
+    filled automatically by the SNS, and will process by Datacube.
```
Comment (alexgleith) [inline dags/landsat-scenes-sync/README.md:22]:
```suggestion
  * This Dag will be started by the scheduler to pull the messages from a second SQS queue, which is 
    filled automatically by the SNS, and will indexed into the Datacube.  
```

File: dags/landsat-scenes-sync/README.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+- landsat-scenes-indexing
+  * This Dag will be started by the scheduler to pull the messages from a second SQS queue, which is 
+    filled automatically by the SNS, and will process by Datacube.  
+
+## Airflow Requirements
+
+Install pip modules
+
+```bash
+    pip install --user stactools[all] && pip install --user pystac && pip install --user
```
Comment (alexgleith) [inline dags/landsat-scenes-sync/README.md:29]:
```suggestion
    pip install --user stactools[all] pystac
```

File: dags/utils/url_request_utils.py [hunk 8]
```diff
@@ -126,6 +126,11 @@ def copy_s3_to_s3(
     :return: None
     """
 
+    # logging.debug(
+    #     f"s3_conn_id {s3_conn_id} source_bucket {source_bucket} destination_bucket {destination_bucket}"
+    #     f" source_key {source_key} destination_key {destination_key} request_payer {request_payer} acl {acl}"
+    # )
+
```
Comment (alexgleith) [inline dags/utils/url_request_utils.py:133]: ???

**Review by alexgleith — CHANGES_REQUESTED** [review]

File: dags/landsat-scenes-sync/README.md (hunk 1 above)
Comment (carioca-au) [reply dags/landsat-scenes-sync/README.md]: Done

File: dags/landsat-scenes-sync/README.md (hunk 2 above)
Comment (carioca-au) [reply dags/landsat-scenes-sync/README.md]: Done

File: dags/landsat-scenes-sync/README.md (hunk 5 above)
Comment (carioca-au) [reply dags/landsat-scenes-sync/README.md]: Done

File: dags/landsat-scenes-sync/README.md (hunk 7 above)
Comment (carioca-au) [reply dags/landsat-scenes-sync/README.md]: Done

File: dags/utils/url_request_utils.py (hunk 8 above)
Comment (carioca-au) [reply dags/utils/url_request_utils.py]: Removed

**Review by alexgleith — APPROVED** [review]

**Pull request merged**